lib/resources/IssuerFraudRecords.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+  path: '',
```

Comment:
Why empty the `path` up here and then pass one in each method? I think you should be able to implement this almost identically to country spec:

``` js
module.exports = StripeResource.extend({

  path: 'country_specs',

  includeBasic: [
    'list', 'retrieve',
  ],
});
```

Reply:
👍 I didn't know list could take a param as I missed the other examples of it. When I implemented a new method, the base path automatically tacks on `/` when anything is appended to it, so I separated it out in order to make sure the `/` didn't get inserted. 

But I'll switch it over to `list` and that should make everything cleaner. Thanks!